apps/remix-ide/src/app/panels/terminal.js

```diff
@@ -746,10 +746,15 @@ class Terminal extends Plugin {
       }
     }
     try {
-      await this.call('scriptRunner', 'execute', script)
+      if (script.trim().indexOf('git') === 0) {
```

Comment:
Keep the same logic everywhere. Use startWith instead of indexOf() (except if you need the index for later, which is not the case here).